src/App.js

```diff
@@ -107,25 +125,50 @@ function App() {
     });
   }, [])
 
+  const printScore = (score) => {
+    const txtColor = score > 0.55 ? "text-black" : "text-red-400";
+    score = String(score).slice(0, 6);
+
+    return (
+      <h2 className={txtColor}>{score}</h2>
+    )
+  }
+
   return (
     <div className="App">
-      <input
-        id="standard-read-only-input"
-        type="text"
-        label="Type your sentences here"
-        onChange={(e) => setText(e.target.value)}
-        value={testText}
-        rows={4}
-        variant="outlined"
-      />
-      <br />
-      <br />
-      {testText !== "" ?
-        <button style={{ width: "20vh", height: "5vh" }} variant="outlined" onClick={() => getSentimentScore(testText)}>Calculate</button>
```

Comment:
Why are you setting styles on the button directly? This is trash. Use tailwind to set width and height. 
What is `variant="outlined"`?

Reply:
😅 Sorry to hear you think that this is trash given that this is your original code 😂